UnicornRender/video/source/Mesh.cpp

```diff
@@ -5,14 +5,25 @@
 */
 
 #include <unicorn/video/Mesh.hpp>
+#include <unicorn/utility/Logger.hpp>
 
 namespace unicorn
 {
 namespace video
 {
-Mesh::Mesh(Material material)
-    : m_material(material)
+Mesh::Mesh() : name("DefaultName"), m_material(nullptr)
+
```

Comment:
why empty line? maybe move the member init to their own lines instead of oneliner